linebot/api.py

```diff
@@ -1148,6 +1149,244 @@ def get_bot_info(self, timeout=None):
+        https://developers.line.biz/en/reference/messaging-api/#get-audience-groups
+
+        :param int page: The page to return when getting (paginated) results. Must be 1 or higher
+        :param str description: The name of the audience(s) to return
+        :param str status: IN_PROGRESS | READY | FAILED | EXPIRED
+        :param int size: The number of audiences per page. Default: 20, Max: 40
+        :param bool include_external_public_group: true | false
+        :param str create_route: How the audience was created.
+        :type create_route: OA_MANAGER | MESSAGING_API
+        :return: AudienceGroup instance
+        """
+        params = dict([(param, eval(param)) for param in
+                       ['page', 'description', 'status', 'size',
+                        'includesExternalPublicGroup', 'createRoute']
+                       if eval(param) is not None])
+        response = self._get(
+            '/v2/bot/audienceGroup/list?',
+            params=params,
+            timeout=timeout
+        )
+        result = []
+        for audience_group in response.json.get('audienceGroups', []):
+            result.append(AudienceGroup.new_from_json_dict(audience_group))
+        if response.json.get('hasNextPage', False):
+            result += self.get_audience_group_list(page + 1, description, status, size,
+                                                   include_external_public_group,
+                                                   create_route, timeout)
+        return result
+
+    def delete_audience_group(self, audience_group_id, timeout=None):
+        """Delete an existing audience.
+
+        https://developers.line.biz/en/reference/messaging-api/#set-description-audience-group
+
+        :param str audience_group_id: The audience ID
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        self._delete(
+            '/v2/bot/audienceGroup/{}'.format(audience_group_id),
+            timeout=timeout
+        )
+
+    def rename_audience_group(self, audience_group_id, audience_group_name, timeout=None):
+        """Modify the name of an existing audience.
+
+        https://developers.line.biz/en/reference/messaging-api/#set-description-audience-group
+
+        :param str audience_group_id: The audience ID
+        :param str audience_group_name: The new audience's name
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        """
+        self._put(
+            '/v2/bot/audienceGroup/{}/updateDescription'.format(audience_group_id),
```

Comment:
Please input property to `{}` 

refer another api: https://github.com/line/line-bot-sdk-python/blob/master/linebot/api.py#L296

Reply:
modified